molecule/model/schema_v2.py

```diff
@@ -105,7 +105,7 @@ def pre_validate_base_schema(env, keep_string):
                                 'schema': {
                                     'password': {
                                         'type': 'string',
-                                        'regex': '^[{$]+[a-z0-9A-Z]+[}]*$',
+                                        'regex': '^[{$]+[a-z0-9A-Z_]+[}]*$',
```

Comment:
I must ask myself why we have this regex in the first place? Can we drop it altogether!?

Reply:
Looks like it forces that value to an environment variable, safe-guarding developers from putting secrets there.  I'm in alignment with you, should be developer's discretion and it slims down the number of test cases if the regex were to be expanded.  If anything a warning in the docs should be enough of a warning about not using secrets in config.

Reply:
Made the update, removing the regex validation.  Failed test seems unrelated.

Reply:
Cool! I'm re-running the failed CI build.

>  If anything a warning in the docs should be enough of a warning about not using secrets in config.

Feel like adding it? We'd also need a changelog entry for this (manual at this point).

Reply:
Changelog and docs addition look good ?

Reply:
Great :+1: